cfpb/cfgov-refresh
# Set eregs_core to be an optional app

eregs_core not being optional was preventing the local server from running if one's `DEPLOY_ENVIRONMENT` var was set to `build` (which mine was, having used it for testing the new Ask CFPB a couple months ago) and one didn't have eregs_core installed.

## Changes

- Adds eregs_core to `OPTIONAL_APPS` instead of `INSTALLED_APPS`

## Testing

1. On master, set your `DEPLOY_ENVIRONMENT` env var to `build` and try to run the server.
1. Observe it failing to find eregs_core.
1. Pull fork
1. Run server

## Checklist
